src/ExadelMentorship.Jobs/ReportJob.cs

```diff
@@ -42,14 +45,29 @@ protected async override Task ExecuteAsync(CancellationToken stoppingToken)
         //System.NotSupportedException: 'Only public methods can be invoked in the background'
         public async Task Execute(ReportUser reportUser)
         {
+            var client = new HttpClient
```

Comment:
Use IHttpClientFactory way. Keep the url in the config.

Reply:
done